### ISSUE TITLE
[youtube] Search results download broken

### ISSUE THREAD

**Issue by hexagonal-sun** [ytdl-org/youtube-dl#27740]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.01.08**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'--verbose', u'https://www.youtube.com/results?search_query=ham+radio']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.01.08
[debug] Python version 2.7.16 (CPython) - Darwin-20.2.0-arm64-arm-64bit
[debug] exe versions: ffmpeg 4.3.1, ffprobe 4.3.1
[debug] Proxy map: {}
[youtube:tab] results: Downloading webpage
ERROR: Unable to recognize tab page; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 803, in wrapper
    return func(self, *args, **kwargs)
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 824, in __extract_info
    ie_result = ie.extract(url)
  File "./youtube-dl/youtube_dl/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "./youtube-dl/youtube_dl/extractor/youtube.py", line 3229, in _real_extract
    raise ExtractorError('Unable to recognize tab page')
ExtractorError: Unable to recognize tab page; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

youtube-dl fails to download a list of videos generated from a youtube search query.  Command line used was:

```
./youtube-dl "https://www.youtube.com/results?search_query=ham+radio"
```

**Comment by aandroide** [comment]
I also confirm to myself error after the update from January 3rd to January 8th

**Comment by aandroide** [comment]
error is: 'NoneType' object has no attribute 'url'

**Comment by pukkandan** [comment]
`youtube:search_url` extractor no longer exists. Use `"ytsearch:ham radio"` instead

**Comment by hexagonal-sun** [comment]
> `youtube:search_url` extractor no longer exists. Use `"ytsearch:ham radio"` instead

Thanks for the info.  In order to get this working I had to use: `ytsearch5:ham radio`.

If this is the way yt searches are to be downloaded, then I would suggest maybe adding a better message to the user so it's not reported as a bug in the future.

**Comment by pukkandan** [comment]
The `YoutubeSearchURLIE` that used to handle search URLs should be reinstated imo

**Comment by dirkf** [comment]
Closed with merge of #27749.
